en/messages.json

```diff
@@ -168,25 +254,59 @@
     "pageInputIconGenerateNewAlias": {
         "message": "Generate New Alias"
     },
+    "pageInputIconGenerateNewAlias_mask": {
+        "message": "Generate new mask"
+    },
+    "pageInputIconGenerateNewRandomMask": {
+        "message": "Generate new random mask"
+    },
+    "pageInputIconAdditionalEmailMasks": {
+        "message": "Additional email masks",
+        "description": "In the website content icon pop-up, users can re-use masks they've already created."
+    },
+    "pageInputIconPreviouslyUsedMask": {
+        "message": "Previously used on this website",
+        "description": "In the website content icon pop-up, users can select specific masks they've created on the current website."
+    },
     "pageInputIconGetUnlimitedAliases": {
         "message": "Get Unlimited Aliases"
     },
+    "pageInputIconGetUnlimitedAliases_mask": {
+        "message": "Get unlimited masks"
+    },
     "pageInputIconUseExistingAliasFromTheSite": {
         "message": "Use Existing Alias from this Site…"
     },
+    "pageInputIconUseExistingAliasFromTheSite_mask": {
+        "message": "Use existing mask from this site…"
+    },
     "pageInputIconRecentAliases": {
         "message": "Recent Aliases"
     },
+    "pageInputIconRecentAliases_mask": {
+        "message": "Recent masks"
+    },
     "pageInputIconMaxAliasesError": {
         "message": "You have already created the maximum number of aliases",
         "description": "Tells the user they have already created the max number of aliases."
     },
+    "pageInputIconMaxAliasesError_mask": {
+        "message": "You have already created the maximum number of masks",
+        "description": "Tells the user they have already created the max number of masks."
+    },
     "pageFillRelayAddressLimit": {
         "message": "You’ve reached the alias limit."
     },
+    "pageFillRelayAddressLimit_mask": {
+        "message": "You’ve reached the mask limit."
+    },
+    "pageNoMasksRemaining": {
+        "message": "You have no more free email masks.",
+        "description": "Error message that appears when a free tier user uses all of their masks."
+    },
     "profilePageInvalidAliasCharactersError": {
         "message": "Character(s) not allowed: $CHARACTERS$",
-        "description": "Error message that appears when the user tries to use special characters when labelling an alias.",
+        "description": "Error message that appears when the user tries to use special characters when labelling an mask.",
```

Comment:
@peiying2 Is it okay for me to update descriptions on previous strings without a new ID change? There's nothing new to translate.

Reply:
Comment update doesn't affect strings. Please go ahead.